php/Controllers/SettingsController.php

```diff
@@ -166,122 +319,247 @@ private function getProfile(): ?ProfileModel {
+    /**
+     * WP Admin tools settings page
+     * @void string HTML for page holding form with setting inputs
+     */
+    public function settings_subpage_tools_cb() {
 
-        $processor_endpoint = 'https://processor.coyote.pics';
+        /*
+         * Return when no profile is set or when in standalone
+         */
+        if (!$this->profile || $this->is_standalone)
+            return;
 
-        $info = __('Using a remote service, your WordPress installation will be queried remotely and this process will populate the associated Coyote organisation. Depending on your WordPress installation, this process may take a while to complete.', COYOTE_I18N_NS);
-        $error = __('If the status of the processing job keeps resulting in an error, consider decreasing the batch size.', COYOTE_I18N_NS);
-        $idempotence = __('This process does not modify your WordPress content itself, and may be used more than once.', COYOTE_I18N_NS);
+        ?>
+        <div class="wrap">
 
-        echo "
-            <p>{$info}</p>
-            <p>{$error}</p>
-            <p>{$idempotence}</p>
-            <div id=\"process-existing-posts\">
-                <div class=\"form-group\">
-                    <label for=\"coyote_processor_endpoint\">" . __('Processor endpoint', COYOTE_I18N_NS) . ":</label>
-                    <input readonly {$process_disabled} id=\"coyote_processor_endpoint\" name=\"coyote_processor_endpoint\" type=\"text\" size=\"50\" maxlength=\"100\" value=\"{$processor_endpoint}\">
-                </div>
+            <h2><?= $this->subpage_title_tools; ?></h2>
+            <form method="post" action="options.php">
 
-                <div class=\"form-group\">
-                    <label for=\"coyote_batch_size\">" . __('Batch size', COYOTE_I18N_NS) . ":</label>
-                    <input id=\"coyote_batch_size\" type=\"text\" size=\"3\" maxlength=\"3\" value=\"{$batch_size}\">
+                <div id="coyote_verify_resource_group_container">
+                    <button class="button button-primary" type="button" id="coyote_verify_resource_group" aria-describedby="coyote_verify_resource_group_hint">Verify resource group</button>
+                    <span role="alert" id="coyote_verify_resource_group_status"></span>
+                    <p id="coyote_verify_resource_group_hint">A resource group is required to make dynamic updates to image description work. When encountering update problems, verify the group exists.</p>
                 </div>
 
-                <div id=\"process-controls\">
-                    <button id=\"coyote_process_existing_posts\" {$process_disabled} type=\"submit\" class=\"button button-primary\">" . __('Start processing job', COYOTE_I18N_NS) . "</button>
-                    <button id=\"coyote_cancel_processing\" {$cancel_disabled} type=\"button\" class=\"button\">" . __('Cancel processing job', COYOTE_I18N_NS). "</button>
+                <hr>
```

Comment:
Please add `aria-hidden="true"` to `<hr>` separators that precede or follow a heading.  They create unneeded noise for screen reader users.

Reply:
Might as well get rid of this `<hr>`, it doesn't add much that we can't do with CSS. 
In @robbinrietdijk's defense - this `<hr>` pre-exists his changes.

Reply:
Removed the <hr>